apps/server/src/modules/authentication/uc/account.uc.ts

```diff
@@ -54,15 +54,17 @@ export class AccountUc {
 		if (limit > 100) {
 			throw new ValidationError('Limit is greater than 100.');
 		}
-		if (!(await this.isSuperhero(currentUser))) {
+		if (!(await this.isSuperhero(currentUser)) && !(await this.isAdmin(currentUser))) {
```

Comment:
Please test if for the permission 'STUDENT_LIST' or 'TEACHER_LIST' instead of the role 'administrator'.
For superhero the school is not relevant. For all other roles it is important to check that the user is at the correct school.

Reply:
The permission is now checked with the hasPermissionsToAccessAccount method.